packages/plugin-stdio/src/StdioPlugin.ts

```
@@ -152,7 +152,7 @@ export class StdioPlugin implements IPlugin, ITransport {
       return;
     }
 
-    const commandModule: CommandModule<T, T> = {
+    const commandModule: CommandModule<{}, T> = {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)